Sources/MeiliSearch/Model/Stat.swift

```diff
@@ -16,6 +16,12 @@ public struct AllStats: Codable, Equatable {
     /// Dictionary of all Indexes containing the stat for each Index.  
     public let indexes: [String: Stat]
 
+    enum CodingKeys: String, CodingKey {
```

Comment:
If I'm not wrong a `CodingKeys` is for re-mapping the keys, only if you want to use a different key name in your code and in the JSON key.
That's not the case here because you are not redefined anything, for example: `case databaseSize = "DBSize"`.

Reply:
That's the result of auto programming code. All  unnecessary`CodingKeys` were removed.